### DISCUSSION TITLE
Cross-Origin-Opener-Policy issue with socials

### DISCUSSION THREAD

**Discussion by julienbrg** [reown-com/appkit#2602] [Q&A]
Hello! 

I upgraded @web3modal/ethers from `v4.1.6` to `v5.0.8` to use the socials in addition to email login, but I'm getting this error at the end of the login workflow (with Google, Twitter and Discord): 

```
Cross-Origin-Opener-Policy policy would block the window.closed call.
```

I also get a little red "Untrusted Origin" toast at the top of the Web3 modal. 

**You can reproduce it here: https://deploy-preview-21--genji-app.netlify.app/**

This is my PR: https://github.com/w3hc/genji/pull/21

I tried to edit my `next.config.js` file, but I can't get this working. It's a pretty basic Next.js app template called [Genji](https://github.com/w3hc/genji) that I often use for quick prototyping. There's probably something missing in my app. 

Thanks for your amazing work!

**Comment by rtomas** [comment]
hi @julienbrg, It's working ok in localhost ?
The issue "Untrusted Origin" it's because you have to whitelist the domain on https://cloud.walletconnect.com/
